SMLHelper/Options/ModConfig.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using Oculus.Newtonsoft.Json;
+
+namespace SMLHelper.V2.Options
+{
+    /// <summary>
+    /// A framework for handling JSON config files for your mods.
+    /// </summary>
+    /// <example>
+    /// <code>
+    /// using System;
+    /// using SMLHelper.V2.Options;
+    /// using SMLHelper.V2.Utility;
+    /// using UnityEngine;
+    /// 
+    /// public class Config : ModConfig {
```

Comment:
Funny thing, I came up with exactly same naming in my code (base class - ModConfig, derived - just Config), but later I switched to the opposite and made Config base class. You use derived name just in couple of places when you declare and load config and base class name is used everywhere else.

Reply:
I was mostly just sticking with the same convention as ModOptions, but this is all likely to be changed soon :)